Formula/jsonnet-bundler.rb

```diff
@@ -0,0 +1,20 @@
+class JsonnetBundler < Formula
+  desc "Package manager for Jsonnet"
+  homepage "https://github.com/jsonnet-bundler/jsonnet-bundler"
+  url "https://github.com/jsonnet-bundler/jsonnet-bundler.git",
+    :tag      => "v0.4.0",
+    :revision => "447344d5a038562d320a3f0dca052611ade29280"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "static"
+    bin.install "_output/jb"
+  end
+
+  test do
+    run_output = shell_output("#{bin}/jb 2>&1")
+    assert_match "A jsonnet package manager", run_output
```

Comment:
Does this test just print the help text?

Reply:
Yes.

Reply:
We need a test that exercises some functionality of the command.

Reply:
Well, I've personally seen examples where formula get away with _**not**_ doing this (https://github.com/Homebrew/homebrew-core/blob/master/Formula/terragrunt.rb), but I digress.

Anyway, the `jb` binary is pretty simple. If you want to use Jsonnet inside of a Git repo, you simply run `jb init`, and get the following file initially:

**jsonnetfile.json**
```json
{
  "version": 1,
  "dependencies": [],
  "legacyImports": true
}
```

From there, you run commands like `jb install https://github.com/grafana/grafonnet-lib/grafonnet` to place the lock the `grafonnet` repository into the current repo. It creates a `vendor/` folder that looks like this:

```shell
$ tree vendor/

vendor
├── github.com
│   └── grafana
│       └── grafonnet-lib
│           └── grafonnet
│               ├── alert_condition.libsonnet
│               ├── alertlist.libsonnet
│               ├── annotation.libsonnet
│               ├── cloudwatch.libsonnet
│               ├── dashboard.libsonnet
│               ├── dashlist.libsonnet
│               ├── elasticsearch.libsonnet
│               ├── gauge.libsonnet
│               ├── grafana.libsonnet
│               ├── graph_panel.libsonnet
│               ├── graphite.libsonnet
│               ├── heatmap_panel.libsonnet
│               ├── influxdb.libsonnet
│               ├── link.libsonnet
│               ├── log_panel.libsonnet
│               ├── loki.libsonnet
│               ├── pie_chart_panel.libsonnet
│               ├── pluginlist.libsonnet
│               ├── prometheus.libsonnet
│               ├── row.libsonnet
│               ├── singlestat.libsonnet
│               ├── sql.libsonnet
│               ├── table_panel.libsonnet
│               ├── template.libsonnet
│               ├── text.libsonnet
│               └── timepicker.libsonnet
└── grafonnet -> github.com/grafana/grafonnet-lib/grafonnet
```

That's the most basic functionality. I'm not really a Ruby expert, so I'm not sure the commands needed to do that. Thinking about it, the tests _should_ be:

- Run `jb init`
- Run `jb install` on a repo like the one above, and test if the `vendor` directory is present.

Do you know how to accomplish that?

Reply:
@SMillerDev Nevermind, I did some Googling, and figured it out. Check the latest push please.